lib/Maintenance/Tasks/CacheCleanupTask.php

```diff
@@ -54,9 +54,8 @@ public function __construct(CoreHandlerInterface $cacheHandler, LoggerInterface
      */
     public function execute()
     {
-        if (!$this->lock->isAcquired() && date('H') <= 4) {
+        if ($this->lock->acquire() && date('H') <= 4) {
```

Comment:
```suggestion
        if (date('H') <= 4 && $this->lock->acquire()) {
```
Should not lock out of time 0:00 and 4:59, or?

Reply:
Yep, makes sense! Actually it shouldn't make any difference in the functionality, since the lock lasts 24 hours, but it's definitely nicer to not call `acquire()` if not necessary 😊

Reply:
Fixed it on the master as well: a46dccf1045abe1126767333f93e9171242b4d1a